### DISCUSSION TITLE
Question about Al2O3 crystal symmetry (primitive cell)

### DISCUSSION THREAD

**Discussion by WenxiLee** [mtex-toolbox/mtex#1963] [Ask Anything] [answered]
Hi all,

We have a dataset labeled with alpha-Al2O3 using the primitive cell convention (with lattice parameters [5.128, 5.128, 5.128, 55.7, 55.7, 55.7]), instead of the trigonal convention ([5.1, 5.1, 13, 90, 90, 120]) as shown in this example: https://mtex-toolbox.github.io/SlipSystemAlumina.html.

I attempted to define the crystal symmetry in MTEX using the command CS=crystalSymmetry('trigonal',[5.128,5.128,5.128],[55.27*degree,55.27*degree,55.27*degree]), but unfortunately, the IPF (Inverse Pole Figure) colors do not match our measurements. I am currently struggling to determine how to correctly define the crystal symmetry in MTEX. Your assistance would be greatly appreciated.

**Comment by lfgmorales** [comment, marked as answer]
Hi

I am not sure what angles are 55.27, but no crystal system (as far as I know) have three equal non-90 or 120 degrees. Note that even in the triclinic case, the angles can be different from 90 or 120 BUT they are differenr from each other. Maybe you need to upload an image for evaluation,

Cheers

Luiz

cubic   a=b=c, α = β = γ = 90°


tetragonal      a = b ≠ c, α = β = γ= 90°
orthorhombic    a ≠ b ≠ c, α = β = γ= 90°


rhombohedral    a = b = c, α = β = γ ≠ 90°

hexagonal       a = b ≠ c, α = β = 90°, γ = 120°




monoclinic      a ≠ b ≠ c, α = γ = 90°, β ≠ 90°

triclinic       a ≠ b ≠ c, α ≠ β ≠ γ





Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 24 Sep 2023, at 03:38, WenxiLee ***@***.***> wrote:

﻿

Hi all,

We have a dataset labeled with alpha-Al2O3 using the primitive cell convention (with lattice parameters [5.128, 5.128, 5.128, 55.7, 55.7, 55.7]), instead of the trigonal convention ([5.1, 5.1, 13, 90, 90, 120]) as shown in this example: https://mtex-toolbox.github.io/SlipSystemAlumina.html.

I attempted to define the crystal symmetry in MTEX using the command CS=crystalSymmetry('trigonal',[5.128,5.128,5.128],[55.27degree,55.27degree,55.27*degree]), but unfortunately, the IPF (Inverse Pole Figure) colors do not match our measurements. I am currently struggling to determine how to correctly define the crystal symmetry in MTEX. Your assistance would be greatly appreciated.

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1963>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN2UISJ7KQQOHIATHRTX36FJFANCNFSM6AAAAAA5ESQXRA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1963", "url": "https://github.com/mtex-toolbox/mtex/discussions/1963", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by WenxiLee** [reply]
Hi Dr. Luiz,

Thank you for your response! Yes, I've managed to figure it out. The relationship between trigonal, hexagonal, and rhombohedral crystal systems is what had me puzzled.
